### PR TITLE
Share 114 implement docker

### PR DESCRIPTION
**Pull request by fladens**
Implementation of Docker

### CODE REVIEW

File: docker/config/parameters.yml.dist [hunk 1]
```diff
@@ -0,0 +1,44 @@
+parameters:
+    database_driver:   pdo_mysql
+    database_host:     db.catroweb.dev
+    database_port:     3306
+    database_name:     symfony
+    database_user:     root
+    database_password: root
+    database_server_v: 10.3.11
```
Comment (HCrane) [inline docker/config/parameters.yml.dist:8]: it should be `database_server_v: mariadb-10.3.11`

File: docker/php/LICENSE [hunk 2]
```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```
Comment (HCrane) [inline docker/php/LICENSE]: Out of curiosity is this license needed?

**Comment by HCrane** [comment]
Something i noticed when trying to do composer updates -> the `composer.json` and `composer.lock` do not get changed so a real update is not done only inside the containers...this should change.

**Review by dmetzner — APPROVED** [review]

**Comment by HCrane** [comment]
Rebased to latest develop, all test run fine natively so we are good to go for merge!

**Review by HCrane — APPROVED** [review]

**Pull request merged**